src/main/java/racingcar/domain/RacingGameRepository.java

```diff
@@ -0,0 +1,13 @@
+package racingcar.domain;
+
+import java.util.List;
+import java.util.Map;
+
+public interface RacingGameRepository {
+
+    int saveAll(List<Cars> cars);
+
+    Map<Integer, Cars> getGameResultSet();
+
+    List<String> getWinners();
+}
```

Comment:
```suggestion
public interface GameRepository<T> {

    int save(List<T> objects);

    Map<Integer, T> getResults();

    List<String> getWinners();
}
```

인터페이스를 좀 더 범용성있게 작성해보는 것은 어떨까요

Reply:
이렇게 보니 꼭  springframework.data.repository 의 CrudRepository가 생각나네요ㅎㅎ 👍